kafka-connector/README.md

```diff
@@ -0,0 +1,48 @@
+# Kafka Connector Resources
+
+This directory contains the OpenShift template and configuration files needed to deploy a KafkaConnector resource using Streams for Apache Kafka. The Connector CR configures the Debezium for Postgresql connector plugin in your Kafka Connect cluster to monitor changes in your database outbox table and publish events to the defined topic.
+
+### Debezium Property Configs
+
+The Kafka Connector CR template provided in this repo, in its default state, is not complete and requires replacing the config section that sets the Kafka config properties for both the Debezium connector and the Outbox event router.
+
+In the `configs` directory are some default base configs that can be used, and are defined separately for each supported Debezium version. We recommend using these configs in combination with the base Debezium template to generate your own deployable template file for consumption in your deployment pipeline or for Ephemeral testing -- See [Using the Templates](#using-the-templates).
+
+The configuration settings are generally the same across multiple versions, but property names have changed in the past, so it is recommended to use the config specific to the version of Debezium you are running.
+
+### Kafka Connector (Debezium Connector)
+
+The KafkaConnector CR template can be used to deploy a Kafka Connector targeting your Kafka Connect cluster and database. The template is designed to allow for multiple service providers to use the same template while avoiding name duplication or conflicts with topics as these Connectors will likely live in the same namespace and target the same Kafka infrastructure.
+
+As mentioned earlier, the template in its default state is not complete, and requires replacing the config section that sets the Kafka config properties for both the Debezium connector and the Outbox event router.
+
+#### Using the Templates
+
+> [!NOTE]
+> This process leverages [yq](https://github.com/mikefarah/yq?tab=readme-ov-file#install) to create the template, you will need to install it
+
+To generate your deployment template:
+1. Determine the Debezium version you are using (must be one of the supported versions in `configs` directory)
```

Comment:
Given the nature of different yaml per versions, we'll probably want to couple this with internal docs on how to add new config files as new versions need to be supported.

Reply:
+1 theres a lot of documentation needs for this subject mentioned in the TODO's, we can lump this into that. Specifically resources on how to find what settings are available and such, which could also be used by SP's wanting to define custom configs too for themselves